ls1intum/Artemis
# `Development`: Disable unevaluated prometheus metrics

### Checklist
#### General



### Motivation and Context
Currently, a lot of auto-generated Metrics are provided through the `/management/prometheus` endpoint (around 5MB per request).
This causes unnecessary load as this data
1. has to be stored in the monitoring system, and
2.  is not used at the moment.

### Description
We only want to include metrics which we actively use in our monitoring setup.
Other values can be re-added once they are included in the setup.

### Steps for Testing
**Note:** Due to an IP-allowlist, it is not possible to test this PR on testing servers.
You have to test it locally.

1. Start the server
3. Navigate to http://localhost:8080/management/prometheus/
4. Check that only the metrics contained in the screenshot are listed - the order may vary but no other metrics should be included
![image](https://user-images.githubusercontent.com/5084100/182931806-3968e9cd-7bc2-4604-8dab-f4d29d7092b2.png)


### Review Progress





#### Code Review
#### Manual Tests
